SPECS/mariner-rpm-macros/macros

```diff
@@ -102,6 +102,27 @@
   LDFLAGS="${LDFLAGS:-%{build_ldflags}}" ; export LDFLAGS ; \
   LT_SYS_LIBRARY_PATH="${LT_SYS_LIBRARY_PATH:-%_libdir:}" ; export LT_SYS_LIBRARY_PATH
 
+# Internal-only.  Do not use.  Expand a variable and strip the flags
```

Comment:
Do you know, if it's possible to undefine `__extension_strip_flags` right after it's used with `%undefine __extension_strip_flags` or `%undefine __extension_strip_flags()`? If so, this way we wouldn't need to worry about anyone accidentally calling it.

Reply:
I'm not entirely sure if that's possible, and I'm not familiar enough with RPM macro expansions to give you a solid answer. Either way, after a bit more thought, I went ahead and removed the "internal use" language inherited from Fedora. I'm not actually too concerned with it being available to spec files at this time.